BACKGROUND.md

```diff
@@ -59,31 +59,41 @@ Africa).
 
 ## Switching between countries
 
-There's a helper script, bin/switch-country.py which, if you've
-set up symlinks and directories correctly (see the top of that
-script) which will let you switch between working on Kenya or
-South Africa with:
+We used to support switching between countries in the development
+environment, but as we're now moving towards country-specific forks
+this is no longer supported. See the next section for details on how
+to construct a development environment for a given country.
 
+## Vagrant
+
+The `Vagrantfile` will set up the Kenyan site by default. You can
+override this by setting `COUNTRY_APP` in the environment, e.g.:
 ```
-bin/switch-country.py kenya
-bin/switch-country.py south-africa
+COUNTRY_APP=south_africa vagrant up
 ```
+Or by creating/updating to relevant variable in `conf/general.yml`
+manually (otherwise this will be created the first time you run 
+`vagrant up`).
 
-The Vagrant box should set up an environment in which this
-works.
+The provisioning script used to download the public database
+dumps which are published, for example,
+[here](https://www.pa.org.za/help/api) for each country and set
+up a database for each.
 
-## Vagrant
+This is no longer the default and can now be done by running, e.g.,
+`bin/vagrant-load-data kenya` from within the vagrant box.
```

Comment:
There doesn't seem to be a `bin/vagrant-load-data` script on this branch, and from what I can see the provisioning does still download the public database dumps. So I think either these docs, or the provisioning script need to be updated 🙂

Reply:
Oops, yes - I removed this. Updated to reflect.